File: 2058-concatenation-of-array/concatenation-of-array.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int n = nums.length;
        int[] arr = new int[n*2];
        for(int i=0;i<n;i++){
            arr[i]=nums[i];
        }
        for(int i=0;i<nums.length;i++){
            arr[n++]=nums[i];
        }
        return arr;
        
    }
}

File: 2058-concatenation-of-array/concatenation-of-array.py
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        ans = [0] * 2 * len(nums)
        j=len(nums)
        for i in range(len(nums)):
            ans[i]=nums[i]
            ans[j]=nums[i]
            i+=1
            j+=1
        return ans

        